src/layers/core/point-density-hexagon-layer/point-density-hexagon-layer.js

```diff
@@ -102,6 +135,13 @@ export default class PointDensityHexagonLayer extends Layer {
     });
   }
 
+  _onPercentileChange() {
+    const [lower, upper] = [this.props.lowerPercentile, this.props.upperPercentile];
```

Comment:
I think I did that to avoid lint max-len error. Need to switch mode between framework contributor and app developer.